src/components/Header/Header.js

```diff
@@ -285,157 +104,90 @@ function Header() {
           }}
           key={id}
         >
-          <MenuItem className={drawerChoice} onClick={logoutHandler}>
-            {to === '/login?logout=true' && (
-              <ExitToAppIcon className={logOutIcon} />
-            )}
-            {label}
-          </MenuItem>
+          <MenuItem className={drawerChoice}>{label}</MenuItem>
         </Link>
       ));
     }
-  };
+  }, [drawerChoice, handleDrawerClose, isLoggedIn, menuHeaders]);
 
-  const getAccountChoices = () => {
-    if (menuHeaders && menuHeaders.length) {
-      return menuHeaders
-        .filter(
-          ({ href }) => href === '/login?logout=true' || href === '/settings'
-        )
-        .map(({ id, href: to, label }) => {
+  const getAccountChoices = useCallback(() => {
+    const logoutHandler = () => Auth.logoff();
```

Comment:
delete, we're going to pass Auth.logoff as a callback in onClick

Reply:
not sure if this is just a stylistic preference, but I prefer to minimize nested arrow functions unless it's a useEffect hook or a functional arrow component

Reply:
resolved in db8dbe8